CactbotOverlay/FateWatcherKo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Machina.FFXIV;
+using Machina.FFXIV.Headers;
+
+namespace Cactbot {
+  class FateWatcherKo : FateWatcher {
+    //
+    // for FFXIV KO version: 5.01
+    //
+    // Latest KO version can be found at:
+    // https://www.ff14.co.kr/news/notice?category=3
+    //
+    private const ushort Server_MessageTypeActorControl143 = 323;
```

Comment:
Why don't you rely on the FFIXV plugin here? It should always have the correct opcode for the current game version (that's what the korean build is for after all).